### PR TITLE
Parse enhanced stereo information from CXSMILES

### PR DESCRIPTION
**Pull request by ricrogz**
#### What does this implement/fix? Explain your changes.
This implements parsing of enhanced stereo information in CXSMILES strings.

### CODE REVIEW

File: Code/GraphMol/SmilesParse/CXSmilesOps.cpp [hunk 1]
```diff
@@ -245,13 +245,60 @@ bool parse_radicals(Iterator &first, Iterator last, RDKit::RWMol &mol) {
...
+  if (first >= last || *first != ':') return false;
+  ++first;
+
+  std::vector<Atom *> atoms;
+  while (first != last && *first >= '0' && *first <= '9') {
+    unsigned int aidx;
+    if (read_int(first, last, aidx)) {
+      Atom *atom = mol.getAtomWithIdx(aidx);
+      if (!atom) {
+        std::cerr << "Atom " << aidx << " not found!" << std::endl;
```
[the remark this answers is not in the document]
Comment (ricrogz) [reply Code/GraphMol/SmilesParse/CXSmilesOps.cpp]: That makes sense. I was lazy and just copied from other parser that used it. I can easily replace both occurrences.

File: Code/GraphMol/SmilesParse/CXSmilesOps.cpp [hunk 2]
```diff
@@ -346,7 +395,7 @@ void processCXSmilesLabels(RDKit::RWMol &mol) {
 void parseCXExtensions(RDKit::RWMol &mol, const std::string &extText,
                        std::string::const_iterator &first) {
   // std::cerr << "parseCXNExtensions: " << extText << std::endl;
-  if (!extText.size() || extText[0] != '|') return;
+  if (extText.empty() || extText[0] != '|') return;
```
[the remark this answers is not in the document]
Comment (ricrogz) [reply Code/GraphMol/SmilesParse/CXSmilesOps.cpp]: np :)

**Comment by greglandrum** [comment]
Thanks @ricrogz, I will merge this after the travis builds finish.

**Comment by ricrogz** [comment]
Sounds good. Thanks Greg!

**Pull request merged**